data_models/crowdstrike_fdr_data_model.py

```diff
@@ -16,6 +16,9 @@ def get_process_name(event):
     # Win = \Device\HarddiskVolume2\Windows\System32\winlogon.exe
     # Lin = /usr/bin/run-parts
     # Mac = /usr/libexec/xpcproxy
+    image_fn = deep_get(event, "event", "ImageFileName")
+    if not image_fn:
+        return "" # Explicitly return an empty string if the key DNE
```

Comment:
That makes sense! I'm in favour of returning `None` instead of something like `<NO_PROCESSNAME>`, since it's likely users may try to do string manipulation on the result. Returning `None` makes it easy for them to determine whether they should proceed or not. I'll make that change